src/components/theme/Anontools/Anontools.jsx

```diff
@@ -3,81 +3,85 @@
  * @module components/theme/Anontools/Anontools
  */
 
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { Menu } from 'semantic-ui-react';
 import { FormattedMessage } from 'react-intl';
 import config from '@plone/volto/registry';
+import { useSelector, shallowEqual } from 'react-redux';
 
 /**
- * Anontools container class.
+ * Anontools function.
+ * @function Anontools
+ * @
+ * @returns {string} Markup of the component.
  */
-export class Anontools extends Component {
-  /**
-   * Property types.
-   * @property {Object} propTypes Property types.
-   * @static
-   */
-  static propTypes = {
-    token: PropTypes.string,
-    content: PropTypes.shape({
-      '@id': PropTypes.string,
+const Anontools = () => {
+  const { token, content } = useSelector(
```

Comment:
Let's talk about how to make this using the façade, but ideally, we would use one hook for the token, other for the content.

Also, let's discuss if we always return a value or an object. Just to make sure that we are consistent.

Reply:
Ohkay ,understood